fiasko_bro/code_validator.py

```diff
@@ -237,11 +236,11 @@ def _run_warning_validators_until(self, failed_error_group_name, arguments):
             )
         return warnings
 
-    def validate(self, solution_repo, original_repo=None, **kwargs):
+    def validate(self, path, original_repo=None, **kwargs):
         self.validator_arguments.update(kwargs)
         self.validator_arguments['whitelists'] = self.whitelists
         self.validator_arguments['blacklists'] = self.blacklists
-        self.validator_arguments['solution_repo'] = solution_repo
+        self.validator_arguments['solution_repo'] = LocalRepositoryInfo(path)
         if original_repo:
             self.validator_arguments['original_repo'] = original_repo
```

Comment:
Now, this is a bug: original_repo is now a string, but LocalRepositoryInfo object is expected here

Reply:
Oops that was bad, sorry should have checked!